src/ui/server/controllers/root/cookies/index.ts

```diff
@@ -60,7 +56,7 @@ export const post = (req: Request, res: Response) => {
     userName: getUserNameFromSession(req.session.user),
     ...singleInputPageVariables({ ...PAGE_VARIABLES, BACK_LINK: backLink, START_ROUTE: startRoute }),
     FIELD: FIELDS[FIELD_IDS.OPTIONAL_COOKIES],
-    submittedValue: req.cookies.optionalCookies,
+    submittedValue: req.cookies.optionalCookies || req.cookies['__Secure-optionalCookies'],
```

Comment:
Great work - not a blocker can be done afterwards - how about a constant for this cookie name?